en/CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contributing guidelines for Chinese-to-English
+
+We are working on translating "Hello Algo" from Chinese to English with the following approach:
+
+1. **AI translation**: Carry out an initial pass of translations using the machine learning translator.
+2. **Human optimization**: Manually refine the machine-generated outputs to ensure authenticity and accuracy.
+3. **Pull request review**: The optimized translation will be doubly checked by the reviewers through GitHub pull request workflow.
```

Comment:
I think "double checked" is better ("The optimized translation will be double checked by the reviewers through GitHub pull request workflow

Reply:
Fixed. Thanks!